deployments/integration/environment

```diff
@@ -16,3 +16,5 @@ _set AZUL_DSS_ENDPOINT "https://dss.integration.data.humancellatlas.org/v1"
 _set azul_grafana_endpoint "https://metrics.dev.data.humancellatlas.org"
 
 _set AZUL_ENABLE_MONITORING 1
+
+_set AZUL_LOG_API_GATEWAY 1
```

Comment:
Let's piggy-back this onto AZUL_ENABLE_MONITORING and not add yet another variable.

Reply:
Done.